modules/parser/src/synchronizers/xviz-stream-buffer.js

```diff
@@ -383,11 +383,10 @@ export default class XVIZStreamBuffer {
     });
 
     for (const streamName in timeslice.streams) {
-      let value = timeslice.streams[streamName];
+      const value = timeslice.streams[streamName];
       if (value === null) {
```

Comment:
We shouldn't need special treatment for `null` if the synchronizer knows how to handle it.

Reply:
actually this was a bug I missed, so thanks.